src/main/java/net/gegy1000/overworldtwo/generator/OverworldTwoChunkGenerator.java

```diff
@@ -193,7 +193,7 @@ protected void sampleNoiseColumn(double[] buffer, int x, int z) {
         } else {
             double left = this.surfaceNoise[0].get(x, z);
             double right = this.surfaceNoise[1].get(x, z);
-            surfaceNoise = MathHelper.lerp(tearNoise, left, right);
+            surfaceNoise = MathHelper.clampedLerp(left, right, tearNoise);
```

Comment:
Hmm, was this causing issues? It should *in theory* always be in the range [0; 1] because of the previous checks

Reply:
oh, I just changed that to replicate vanilla, I forgot about the previous checks 😅